scripts/a11y/process_a11y.py

```diff
@@ -0,0 +1,144 @@
+import csv
+import json
+
+from collections import defaultdict
+from statistics import mean
+
+
+class A11yProcessor(object):
+    ERRORS = {
+        '1_1': 'Alt Tag Errors',
+        '1_3': 'Form Errors',
+        '1_4': 'Color Contrast Errors',
+        '4_1': 'HTML Attribute Errors'
+    }
+
+    BRANCHES = {
```

Comment:
@brendansudol I worry a bit about how this is set up ... I believe we could have many agency names not be listed in this `dict` ... the point is that anything not listed should default to `executive`. cc @konklone